docs/firefly-api-code-examples.md

```diff
@@ -0,0 +1,132 @@
+## Code Examples Using Firefly Tools
+
+- [Image FITS Viewer](image-fits-viewer)
+- [FITS Viewers in a group](fits-viewers-in-a-group)
+- [Tracking the mouse](tracking-the-mouse)
+- [Tracking the selection](tracking-the-selection)
+
+###Starting Firefly Tools in JavaScript
+Firefly Tools are loaded when you load`firefly_loader.js` from a server of your choice. For example, to load Firefly Tools from your local server, include the following declaration in your HTML file:
+
+```html
+ <script type="text/javascript" language="javascript" src="http://localhost:8080/fftools/firefly_loader.js">   
+```
+
+Your html document must define some divs, where you will load the viewer widgets, and `onFireflyLoaded()` function.
+
+When Firefly Tools completes loading, it calls `onFireflyLoaded()` JavaScript function. This is where you create Firefly components and place them into HTML `<div>` elements with specified IDs. 
+
+The following are several examples of what can be done in the `onFireflyLoaded()` function. All test data can be found at `http://web.ipac.caltech.edu/staff/roby/demo`. 
+
+###Image FITS Viewer
+Create an image viewer and place it into the `<div>` id `plotHere`.
+
+```html
+<div id="plotHere" style="width: 350px; height: 350px;"></div>
+```
+
+```js
+function onFireflyLoaded() {
+    firefly.showImage('plotHere', {
+        URL      : 'http://web.ipac.caltech.edu/staff/roby/demo/wise-m31-level1-3.fits',
+        Title    : 'Example FITS Image',
+        ColorTable : 16,
+        RangeValues : firefly.util.image.RangeValues.serializeSimple('Sigma',-2,8,'Linear')
+    });
+```
+
+
+###FITS Viewers in a group
+In this example, we create four image viewers. Each belong to the same group `wise-group`. We then set some global parameters so all the plots display the same. Now we plot each of the four images by specifying the URL of the FITS file. By doing this, all the plotting controls will work on all four images simultaneously. 
+
+
+```html
+  <div style="white-space: nowrap;">
+    <div id="w1" style="display:inline-block; width: 250px; height: 250px; border: solid 1px;"></div>
+    <div id="w2" style="display:inline-block; width: 250px; height: 250px; border: solid 1px;"></div>
+    <div id="w3" style="display:inline-block; width: 250px; height: 250px; border: solid 1px;"></div>
+    <div id="w4" style="display:inline-block; width: 250px; height: 250px; border: solid 1px;"></div>
+  </div>
+```
+
+```js
+ function onFireflyLoaded() {
+   firefly.setGlobalImageDef({ ZoomType    : 'TO_WIDTH',
+                                    ColorTable  : 8,
+                                    ZoomToWidth : 250 });   
+   w1.showImage('w1', { URL  : 'http://web.ipac.caltech.edu/staff/roby/demo/wise-m51-band1.fits',
+             Title: 'Wise band 1', plotGroupId : 'wise-group' });
```

Comment:
>   Title: 'Wise band 1', plotGroupId : 'wise-group' });

for all the titles with WISE data, lets use all caps WISE.